stmhal/usbd_conf.c

```diff
@@ -60,15 +60,15 @@ void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd)
   {
     /* Configure USB FS GPIOs */
     __GPIOA_CLK_ENABLE();
-    
+

```

Comment:
Please, no cosmetic changes in this PR, just functional ones related to the F429.